### ISSUE TITLE
Remove xrefs from modules

### ISSUE THREAD

**Issue by bergerhoffer** [openshift/openshift-docs#41954]
In reviewing a module, I noticed that there was an xref in it. A quick search of the modules/ directory showed quite a few in there.

We currently don't allow xrefs in modules per our guidelines. We should do a sweep and remove these xrefs. (Comment out or otherwise leave a note so that they can be found/added back in later, if/when we do allow them again).

**Comment by peterroth** [comment]
Is this still a valid ticket, @bergerhoffer?

**Comment by bergerhoffer** [comment]
@peterroth yes it is still valid, but unfortunately is not a quick/easy fix. They don't need to just be removed, they will each need to be evaluated and see if we need to add them in as a workaround elsewhere. I think we likely have a few hundred :/

**Comment by peterroth** [comment]
I see, @bergerhoffer. In this case I can't fix the ticket because I don't have the reliable resource to build and test the changes.

**Comment by bergerhoffer** [comment]
> I see, @bergerhoffer. In this case I can't fix the ticket because I don't have the reliable resource to build and test the changes.

Agreed. This one will likely need to be done by our team, and likely broken up for several writers to do. 

There are likely other tickets that are more suited for anyone to pick up, if you're looking to contribute more. Let me know if you need any help finding something, if you are!
